### DISCUSSION TITLE
Smallest distribution capable of running YT-DLP

### DISCUSSION THREAD

**Discussion by upintheairsheep** [utmapp/UTM#4208] [UTM Help (iOS)]
What is the smallest file size Linux distribution capable of running YT-DLP? iSH and A-Shell won’t install due to a encryption dependency problem with C. I need to run tubeup, a program for archiving online videos specifically.

**Comment by ktprograms** [comment]
Alpine Linux is one of the lightest, but because it uses musl libc it might not be compatible with your program.

You can also use Debian with no GUI.

**Reply by upintheairsheep** [reply]
I used iSH and it didn’t work. I’ll try Debian. How do you boot with and without gui
